index.js

```diff
@@ -75,6 +75,11 @@ QUnit.notifications = function( options ) {
       var toolbar      = document.getElementById( "qunit-testrunner-toolbar" );
       if ( !toolbar ) { return; }
 
+      var checkboxContainer = toolbar.querySelector(".qunit-url-config");
+      if (!checkboxContainer) { // qunit < 2
```

Comment:
Make a condition to make it backward compat.
As I can see qunit@1 has the same ".qunit-url-config" element. So probably we should do this un-conditionally.